wfe/web-front-end.go

```diff
@@ -392,6 +392,8 @@ func (wfe *WebFrontEndImpl) sendError(response http.ResponseWriter, msg string,
 		fallthrough
 	case http.StatusBadRequest:
 		problem.Type = core.MalformedProblem
+	case http.StatusGone:
+		fallthrough
```

Comment:
It's not totally clear to me that ServerInternalProblem is the right thing to return here.  I guess maybe, because the reason the cert is dead is that something happened in the CA, not because of anything the client did.